ui/analyse/src/study/inviteForm.ts

```diff
@@ -71,6 +75,7 @@ export function view(ctrl: ReturnType<typeof makeCtrl>): VNode {
                   input.value = '';
                   ctrl.invite(v.name);
                   ctrl.redraw();
+                  ctrl.recentlyInvited.setKey(v.name, null);
```

Comment:
A bit of a hack: 

If we put this call inside of `invite` then there is a brief moment where the user is in local storage but not yet invited to the study. The result is the user being suggested (since they're not a member but are in local storage) for a brief instant before disappearing which is pretty jarring. 

The best solution I've found to this is to put this call *after* the `redraw` but lmk if there's a better way